src/environment/environment.production.ts

```diff
@@ -6,6 +6,8 @@ export const environment: EnvironmentFile = {
   daysToCacheJournalData: 7,
   daysToCacheLogs: 7,
   logsPostApiKey: '',
+  logsApiUrl: 'https://prod.mes.dev-dvsacloud.uk/v1/logs',
```

Comment:
The URL is incorrect for PROD but the bigger issue here is that we probably don't want to put URLs to prod env in github at all.

Given the fact that this info comes form MDM anyways - I'd prefer to leave this empty - at least for prod/uat/perf/staging(?) - is there any value of hardcoding those URLs on higher ENVs than local/dev?
If we really need those URLs documented somewhere confluence would be a better place for it imho.

Reply:
I can remove this (and will)   - there is a url in there for the config service already though.